webxrlayers-1.bs

```diff
@@ -775,18 +802,43 @@ is initialized.
 dictionary XRProjectionLayerInit {
   XRTextureType textureType = "texture";
   GLenum colorFormat = 0x1908; // RGBA
-  GLenum? depthFormat;
+  GLenum depthFormat = 0x1902; // DEPTH_COMPONENT
```

Comment:
What do you mean by "the default"?
You are right that for WebGL1, this needs the extension. This was already the case in the previous approach with the booleans because projection layers were always created with a depth buffer.

Reply:
In the previous approach the depth attachment could be supplied by using a Renderbuffer, which we can't do here. I'd feel more comfortable is the default was no depth buffer.

Reply:
You can still do that. Nothing has changed in that regard.
If you pass a value of `0`, no depth buffer is created and you can create your own. I agree that feels a bit weird but I think a projection layer should have a depth buffer by default.